src/containers/aspect/AspectDashboardContainer/AspectDashboardContainer.tsx

```diff
@@ -240,6 +242,8 @@ const AspectDashboardContainer: FC<AspectDashboardContainerProps> = ({
       },
     });
 
+  const aspectUrl = buildAspectUrl({ hubNameId, challengeNameId, opportunityNameId, calloutNameId, aspectNameId });
```

Comment:
isn't `aspectUrl` provided from the props? - I think the one in the props was a mistake.

Reply:
I don't know what you mean with this.
At the top of this file there's the `Provided` interface which is what this Container 'provides'
The aspectUrl in the props of AspectDashboardView is required by the view. 
The Aspects implementation is a bit ugly imo, in this case we have two interfaces `AspectDashboardViewProps` and `Provided` in this file, that match in a lot of properties but are not the same...

Reply:
I got confused - it wasn't clear to me what `ContainerPropsWithProvided` does.